jcl/src/com.ibm.management/share/classes/com/ibm/lang/management/RuntimeMXBean.java

```diff
@@ -28,6 +28,31 @@
 public interface RuntimeMXBean extends java.lang.management.RuntimeMXBean {
 
 	/**
+	 * enum type defines the different states of the VM Idle 
```

Comment:
The constant values reflect J9VMRuntimeStateListener.vmRuntimeState from the VM and the VM constants J9VM_RUNTIME_STATE_ACTIVE and J9VM_RUNTIME_STATE_IDLE. These values need to match the VM values. Please include this information in the comment. Also add a comment above the constants in j9nonbuilds.h to mention the values are reflected here in the Java class library code.

Reply:
I put my feedback in the wrong place. The javadoc comment shouldn't be udpated, but an internal comment added below about the enum value declarations.